cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -113,12 +113,19 @@ public ExecutorState state() {
   }
 
   /**
-   * Kick off the execution.
+   * Pause the load monitor and kick off the execution.
+   *
+   * @param loadMonitor Load monitor.
    */
   public void startExecution(LoadMonitor loadMonitor) {
+    if (loadMonitor != null) {
```

Comment:
If we do not expect the load monitor to be null, can we throw `IllegalArgumentException` and pass a mock object in the unit test?